firmware/application/external/doom/ui_doom.cpp

```diff
@@ -9,6 +9,11 @@
 #include "ui_doom.hpp"
 #include "ui.hpp"
 
+// Provide PROGMEM compatibility for embedded system

```

Comment:
I don't know about that optimisation, except that it should be having it's own Pull Request and not be mixed with the 'Scanning archivtecture' PR